src/pages/Patients/Patients.js

```diff
@@ -330,32 +342,37 @@ const Patients = () => {
         </DateAndTimeWrap>
       </InfoWrapper>
       <div className="dashboard-header mb-2 d-flex justify-content-between flex-wrap w-100">
-        {isShowPatientInfoColumn && (
-          <PatientInfoColumn>
-            <InfoValue>{patients?.length ?? 0} active patients</InfoValue>
-            <div className="d-flex justify-content-between">
-              <StatusIndicator status={riskLevel} size={12} />
-              <Select
-                value={riskLevel}
-                onChange={handleRiskLevelChange}
-                onBlur={handleRiskLevelChange}>
-                {patientRiskData?.map((risk) => {
-                  return (
-                    <option
-                      key={risk.riskType}
-                      value={risk.riskType}
-                      className="select-options">
-                      {risk.riskType} ({risk.numberOfCases})
-                    </option>
-                  );
-                })}
-              </Select>
-            </div>
-          </PatientInfoColumn>
-        )}
+        <PatientInfoColumn>
+          <InfoValue>
+            {totalPatients
+              ? dischargedPatients
+                ? totalPatients - dischargedPatients
+                : totalPatients
```

Comment:
let's calculate this value and store in state in the function where we are fetching the data. that way we will only have to run this calculation once

Reply:
resolved